### PR TITLE
Adding carthage framework

### PR DESCRIPTION
**Pull request by FranDepascuali**
## Summary ##

Some modifications for the framework to be used by https://github.com/Wolox/wolmo-authentication-ios. Includes some errors unseen which we didn't catch from CR.

## Trello card ##
https://trello.com/c/DCeC6592/11-integrar-core-para-poder-ser-utilizado-en-otros-frameworks

### CODE REVIEW

**Comment by woloxci** [comment]
:+1: There are no linter violations.

File: Core/Extensions/Foundation/NSBundle.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ public extension NSBundle {
      - seealso: loadNib(nibName: String)
```
Comment (danielaRiesgo) [inline Core/Extensions/Foundation/NSBundle.swift]: Remove the `see also`

**Comment by danielaRiesgo** [comment]
👍

**Comment by guidomb** [comment]
👍

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Comment by guidomb** [comment]
👍

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Pull request merged**